tooling/templatize/internal/end2end/e2e_test.go

```diff
@@ -82,9 +82,9 @@ func TestE2EKubernetes(t *testing.T) {
 	tmpDir := t.TempDir()
 
 	e2eImpl := newE2E(tmpDir)
-	e2eImpl.AddStep(types.NewShellStep("test", "kubectl get namespaces").WithAKSCluster("dev-svc"), 0)
+	e2eImpl.AddStep(types.NewShellStep("test", "kubectl get namespaces").WithAKSCluster("dev-westus3-svc-1"), 0)
 
-	e2eImpl.SetConfig(config.Configuration{"defaults": config.Configuration{"rg": "hcp-underlay-dev-svc"}})
+	e2eImpl.SetConfig(config.Configuration{"defaults": config.Configuration{"rg": "hcp-underlay-dev-westus3-svc"}})
```

Comment:
Looks like it should work.  Maybe want to define a location const in `e2e.go` though?  I see `westus3` hard-coded in a few places there too.

Reply:
I'd like to see if we can make this dynamic instead, ie use whatever is in `config/`.  It's annoying to have to change this if we decide to update the resource group / aks cluster naming schema or if we want to run this in another environment.